includes/class-wc-calypso-bridge-ecommerce-admin-menu.php

```diff
@@ -28,6 +28,8 @@ class Ecommerce_Atomic_Admin_Menu extends \Automattic\Jetpack\Dashboard_Customiz
 	public function __construct() {
 		parent::__construct();
 		add_action( 'admin_menu', array( $this, 'maybe_hide_payments_menu' ), 10 );
+		add_action( 'admin_menu', array( $this, 'maybe_remove_customizer_menu' ), PHP_INT_MAX );
```

Comment:
Perhaps using _PHP_INT_MAX_ is unnecessary. I expect 3pd plugins to store some settings in the customizer and they may need a way to support the WX environment by force-showing it again. I'd suggest a more reasonable prio like `9999`